src/main/java/seedu/address/logic/commands/LastContactCommand.java

```diff
@@ -1,74 +1,53 @@
 package seedu.address.logic.commands;
 
-import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static java.util.Objects.requireNonNull;
 
-import java.util.List;
+import java.time.LocalDateTime;
+import java.util.Comparator;
 
-import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.LastContact;
+import seedu.address.model.person.IsLastContactedPredicate;
```

Comment:
I think since our lastcontact is a field, perhaps it is better to name the predicate as HasLastContactPredicate? IsLastContacted others may misinterpret as 'is this person the last one I contacted'. Just some food for thoughts hehe 😄